docs/developer/installation.md

```diff
@@ -0,0 +1,66 @@
+
+# Installation
+
+SedTRAILS (Sediment TRAnsport vIsualization and Lagrangian Simulator). 
+================
+This is the step-by-step process to install SedTRAILS.
+
+ 
+:On Windows:
+
+1. Make sure Python (3.13 or newer) and a code editor (e.g., Visual Studio) is installed.
+
+2. Clone the SedTRAILS repository from github in your desired directory:
+
+```bash
+cd <your desired directory>
+git clone https://github.com/sedtrails/sedtrails.git
+```
+
+3. Change the directory:
+
+```bash
+cd sedtrails
+```
+4. Install the dependencies:
+
+
+```bash
+pip install -e .[dev]
+```
+For more details, see the following page:
+https://github.com/sedtrails/sedtrails/blob/dev/CONTRIBUTING.md
+
+
+Confirming the installation
+-----------
+
+Type the following command in the terminal:
+```bash
+sedtrails -v
+```
+
+For help:
+```bash
+sedtrails -h
+
+```
+Running an example model
+-----------
+1. Download data-set from the following link:
+https://surfdrive.surf.nl/files/index.php/s/VUGKZm7QexAXuD9?path=%2Fdfm
```

Comment:
In Markdown you can format hyperlinks as:

```markdown
[hyperlink text](link)
```

Which results in:

[hyperlink text](link)